src/drakvuf.cpp

```diff
@@ -214,13 +214,35 @@ drakvuf_c::drakvuf_c(const char* domain,
     addr_t kpgd,
     bool fast_singlestep,
     uint64_t limited_traps_ttl,
+    const std::set<uint64_t>& ignored_processes_,
     bool libdrakvuf_get_userid)
     : leave_paused{ leave_paused }
     , format{ output }
 {
-    if (!drakvuf_init(&drakvuf, domain, json_kernel_path, json_wow_path, libvmi_conf, kpgd, fast_singlestep, limited_traps_ttl, libdrakvuf_get_userid))
+    GSList* ignored_processes = NULL;
+    for (auto pid : ignored_processes_)
+    {
+        gconstpointer key = GUINT_TO_POINTER(pid);
```

Comment:
No need to make it into a gconstpointer and have a separate variable on the stack.

Reply:
OK